src/GraphQL.Tests/Introspection/IntrospectionResult.cs

```diff
@@ -2,7 +2,1152 @@ namespace GraphQL.Tests.Introspection
 {
     public class IntrospectionResult
     {
-        public static readonly string Data =
+        public static readonly string DataWhenSystemTextJson =
```

Comment:
What is the difference between `DataWhenSystemTextJson` and `DataWhenNewtonsoftJson` ?

Reply:
S.T.Json turns ` into \u0060 (from memory) and ' into another \u1234 and so on. It's a painful difference, but wasn't sure what else to do, as this is a pretty nice, comprehensive test for the serializers that I wanted to do a Theory for to cover both.